### PR TITLE
Feature/add strelka

### PR DESCRIPTION
**Pull request by calchoo**
Added strelka as a supported tool.

### CODE REVIEW

File: docs/source/glossary.rst [hunk 1]
```diff
@@ -105,6 +105,9 @@ General Terms
     Manta
         Manta is an SV caller. Source for Manta can be found `here <https://github.com/Illumina/manta>`__ [Chen-2016]_
 
+    Strelka
+        Strelka is an SV caller. Source for Strelka can be found `here <https://github.com/Illumina/strelka>`__ [Saunders-2012]_
```
Comment (creisle) [inline docs/source/glossary.rst]: I wouldn't say its an SV caller, perhaps saying it is a "SNV and small indel caller" would be more appropriate. Also add a note here that we do not process the SNVs

File: docs/source/supported_dependencies.rst [hunk 2]
```diff
@@ -120,7 +120,10 @@ MAVIS also supports a :ref:`general VCF input <general-vcf-inputs>`. It should b
         - ``1.4.8 (custom)``
         - ``{indels/events_novel_exons,fusions/*}.tsv``
         - [Robertson-2010]_
-
+    *   - :term:`Strelka`
+        - ``1.0.6``
+        - ``passed.somatic.indels.vcf``
```
Comment (creisle) [inline docs/source/supported_dependencies.rst]: Is the file suffix convention something that strelka always produces? or something that we just name the file normally? would someone else running strelka from scratch get a file with this naming convention? is this a suffix or the whole file name? if it is just a suffix you should add the glob asterisk in front

File: mavis/tools.py [hunk 3]
```diff
@@ -478,15 +478,22 @@ def _convert_tool_output(input_file, file_type=SUPPORTED_TOOL.MAVIS, stranded=Fa
...
             ])
-    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ]:
+    elif file_type in [SUPPORTED_TOOL.DELLY, SUPPORTED_TOOL.MANTA, SUPPORTED_TOOL.PINDEL, SUPPORTED_TOOL.VCF, SUPPORTED_TOOL.BREAKSEQ, SUPPORTED_TOOL.STRELKA]:
         rows = []
         vfile = VariantFile(input_file)
         try:
             vfile.header.info.add('END', number=1, type='Integer', description='End of the interval')
         except ValueError:
             pass
         for vcf_record in vfile.fetch():
```
Comment (creisle) [inline mavis/tools.py]: how are you filtering out the SNVs?

File: tests/unit/test_tool.py [hunk 4]
```diff
@@ -712,10 +711,53 @@ def test_insertion(self):
         self.assertEqual(False, bpps[0].opposing_strands)
 
 
+class TestStrelka(unittest.TestCase):
```
Comment (creisle) [inline tests/unit/test_tool.py]: can you add an SNV test here as well to show that we are dropping them?

File: mavis/tools.py (hunk 3 above)
Comment (calchoo) [reply mavis/tools.py]: I assumed we would only use the indel from Strelka as input...

File: docs/source/supported_dependencies.rst (hunk 2 above)
Comment (calchoo) [reply docs/source/supported_dependencies.rst]: I'm pretty sure "passed.somatic.indels.vcf" is a standard output file of strelka? I haven't run the strelka pipeline myself to know for sure.

File: docs/source/glossary.rst (hunk 1 above)
Comment (calchoo) [reply docs/source/glossary.rst]: ok I'll change it

File: tests/unit/test_tool.py (hunk 4 above)
Comment (calchoo) [reply tests/unit/test_tool.py]: sure, but I was assuming we would only indels from Strelka as input

File: mavis/tools.py (hunk 3 above)
Comment (creisle) [reply mavis/tools.py]: ya, if the splitting of indels and snvs is done by strelka then you can ignore my SNV related comments. If it is not done by default though then we will need to worry about verbosely filtering them

**Pull request merged**